src/pyeantic/pyeantic/cppyy_eantic.py

```diff
@@ -139,11 +139,29 @@ def py_make_renf_elem_class(*args):
         if isinstance(v, eantic.renf_class):
             return eantic.renf_elem_class(v)
         else:
+            v = sage_to_gmp(v)
             return eantic.make_renf_elem_class(v)
     elif len(args) == 2:
         K, v = args
         if isinstance(v, (tuple, list)):
             v = cppyy.gbl.std.vector[int](v)
```

Comment:
What about a vector of Sage integers/rationals?

Reply:
True.

Reply:
When you say "vector" you mean "Python list"?

Reply:
Yes. For now I am not sure there is a need to support something more general than tuple/list.